README.md

```diff
@@ -82,16 +82,16 @@ At this time, only the output variables of air_temperature, surface_pressure (co
 Usage: <converter.py> -i INPUT_FILE(S) -o OUTPUT_FILE -d YYYYMMDDHH
 ```
 
-## chem
+## compo
```

Comment:
If changing the name here, should change the directory name to avoid confusion.

Reply:
done, there was also chem in makelists, i corrected it as well.